Formula/frei0r.rb

```diff
@@ -1,8 +1,8 @@
 class Frei0r < Formula
   desc "Minimalistic plugin API for video effects"
   homepage "https://frei0r.dyne.org/"
-  url "https://files.dyne.org/frei0r/releases/frei0r-plugins-1.4.tar.gz"
-  sha256 "8470fcabde9f341b729be3be16385ffc8383d6f3328213907a43851b6e83be57"
+  url "https://files.dyne.org/frei0r/frei0r-plugins-1.6.1.tar.gz"
```

Comment:
Please file an issue with upstream (https://github.com/dyne/frei0r/issues) requesting this be copied into their releases folder and 1.6.1 marked as latest in the `00_LATEST_IS_*` file otherwise the URL will 404 on us as soon as they upgrade.